src/renderer/widgets/CreateWallet/model/form-model.ts

```diff
@@ -71,6 +73,25 @@ const $multisigAlreadyExists = combine(
   { skipVoid: false },
 );
 
+const $hiddenMultisig = combine(
+  {
+    hiddenWallets: walletModel.$hiddenWallets,
+    multisigAccountId: $multisigAccountId,
+    formValues: $createMultisigForm.$values,
+  },
+  ({ multisigAccountId, hiddenWallets, formValues: { chain } }) =>
+    walletUtils.getWalletFilteredAccounts(hiddenWallets, {
+      walletFn: (w) => walletUtils.isMultisig(w),
+      accountFn: (multisigAccount) => {
+        const isSameAccountId = multisigAccount.accountId === multisigAccountId;
+        const chainId = (multisigAccount as MultisigAccount).chainId;
+        const isSameChainId = !chainId || chainId === chain.chainId;
+
+        return isSameAccountId && isSameChainId;
+      },
+    }) || null,
+);
```

Comment:
Or even alter `getWalletFilteredAccounts` to return `null` instead of `undefined` 🤔 
```suggestion
  ({ multisigAccountId, hiddenWallets, formValues: { chain } }) => {
    const multisig = walletUtils.getWalletFilteredAccounts(hiddenWallets, {
      walletFn: (w) => walletUtils.isMultisig(w),
      accountFn: (multisigAccount) => {
        const isSameAccountId = multisigAccount.accountId === multisigAccountId;
        const chainId = (multisigAccount as MultisigAccount).chainId;
        const isSameChainId = !chainId || chainId === chain.chainId;

        return isSameAccountId && isSameChainId;
      },
    });

  return multisig || null;
);
```